js/Game.js

```diff
@@ -7,38 +7,42 @@ class Game {
   }
 
   init() {
-    game.players = Object.assign({}, domUpdates.getPlayerNames());
+    this.players = Object.assign({}, domUpdates.getPlayerNames());
     domUpdates.clearInputs();
     domUpdates.goToGameScreen();
-    return game.players;
+    return this.players;
   }
 
   startRound() {
```

Comment:
Similar concept here, I would expect a method named startRound to do some round initialization work, but not necessarily return it. Usually methods that return a value are named as nouns, rather than verbs. You could refactor this simply by renaming the method to something like newRound